applications/asset_tracker_v2/src/cloud/cloud_codec/aws_iot_codec.c

```diff
@@ -36,6 +65,11 @@ int cloud_codec_decode_config(char *input, struct cloud_data_cfg *data)
 		return -ENOENT;
 	}
 
+	if (has_message_been_handled(root_obj)) {
+		err = -ENOMSG;
```

Comment:
```suggestion
		err = -ECANCELED;
```
Would be more meaningful, after all the message is there.

Reply:
Good suggestion. Its always troublesome to find an errno code that fits the use case at hand.